README.md

```diff
@@ -35,16 +35,16 @@ $ babel script.js --presets airbnb
 ### Via Node API
 
 ```javascript
-require("babel-core").transform("code", {
+require("@babel/core").transform("code", {
   presets: ["airbnb"]
 });
 ```
 
 ### Targeting Environments
 
-This module uses babel-preset-env to target specific environments.
+This module uses @babel/preset-env to target specific environments.
 
-Please refer to [babel-preset-env#targets](https://github.com/babel/babel-preset-env#targets) for a list of available options.
+Please refer to [@babel/preset-env#targets](https://github.com/babel/website/blob/master/docs/preset-env.md#targets) for a list of available options.
```

Comment:
Wouldn't it be better to link to [the website](https://babeljs.io/docs/en/babel-preset-env#options) instead of the website source?

Reply:
Oops, yeah that would be better.